src/main/java/com/example/securitynew/repository/criteria/book/BookSpecificationProviderManager.java

```diff
@@ -19,4 +19,5 @@ public SpecificationProvider<Book> getSpecificationProvider(String key) {
                         new RuntimeException("Can't find correct specification provider for key "
                                 + key));
     }
+
```

Comment:
Remove all SpecificationProvider-, Criterea- and so on.
Because we are using the SPring boot jpa